robot_skills/src/robot_skills/world_model_ed.py

```diff
@@ -174,7 +174,7 @@ def get_entity_info(self, id):
 
     def reset(self, keep_all_shapes=True):
         try:
-            self._ed_reset_srv(keep_all_shapes=keep_all_shapes)
+            return self._ed_reset_srv(keep_all_shapes=keep_all_shapes)
         except rospy.ServiceException, e:
             rospy.logerr("Could not reset ED: {0}".format(e))
```

Comment:
In case of the excption, it should return False.

Reply:
It originally did not return anything, but I'll change it anyway

Reply:
Old code is not always good code